docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html

```diff
@@ -0,0 +1,62 @@
+<nav class="bg-white p4 mn2">
+  <div class="row">
+    <div class="col-fit h8 row-end">
+      <div class="h6 row-center-x">
+        {% render '@icon-square--home-active-large'%}
+      </div>
+    </div>
+    <div class="col-fit h8 row-end">
+      <div class="h6 row-center-x sans-s6 fw-300 lightgray lh-single mx105">
+        /
+      </div>
+    </div>
+    <div class="col-fit h8 row-end">
+      <div>
+        <div class="h2 sans-s1 fw500 lightgray uppercase ls-loose">
+          Organization
+        </div>
+        <div class="h6 row-center-x">
+          {% render '@button-trigger--base-outline', {buttonText: 'epa-journeyman'}, true %}
+        </div>
+      </div>
+    </div>
+    <div class="col-fit h8 row-end">
+      <div class="h6 row-center-x sans-s6 fw-300 lightgray lh-single mx105">
+        /
+      </div>
+    </div>
+    <div class="col-fit h8 row-end">
+      <div>
+        <div class="h2 sans-s1 fw500 lightgray uppercase ls-loose">
+          Space
+        </div>
+        <div class="h6 row-center-x">
+          {% render '@button-trigger--base-outline', {buttonText: 'dev'}, true %}
+        </div>
+      </div>
+    </div>
+    <div class="col-fit h8 row-end">
+      <div class="h6 row-center-x sans-s6 fw-300 lightgray lh-single mx105">
+        /
+      </div>
+    </div>
+    <div class="col-fit h8 row-end">
+      <div>
+        <div class="h2 sans-s1 fw500 uppercase ls-loose">
+          App
+        </div>
+        <div class="row-center-x">
+          <div class="col-fit mr2 h6 row-center-x">
+            {% render '@icon-square--app-ok-large'%}
```

Comment:
I'd suggest removing the sizing here `large` in favor of letting the block the icon is in define the size.